packages/plugin-conversation/src/conversation.js

```diff
@@ -561,7 +561,7 @@ const Conversation = SparkPlugin.extend({
       kmsMessage: {
         method: `create`,
         uri: `/resources`,
-        userIds: params.participants,
+        userIds: cloneDeep(params.participants),
```

Comment:
Before this change userIds held a reference to params.participants. This mean that any changes in params.participants was also reflected here leading so some unexpected side effects. With this change kms.userIds will have its own array of user ids